opentelemetry-user-events-logs/src/logs/exporter.rs

```diff
@@ -362,10 +360,12 @@ impl Debug for UserEventsExporter {
 
 impl opentelemetry_sdk::logs::LogExporter for UserEventsExporter {
     async fn export(&self, batch: opentelemetry_sdk::logs::LogBatch<'_>) -> OTelSdkResult {
+        let mut res = Ok(());
         for (record, instrumentation) in batch.iter() {
-            let _ = self.export_log_data(record, instrumentation);
+            // no, res is not overwritten as the batch will only have one item
```

Comment:
If batch is always once, loop can be avoided.
```rust
let (record, instrumentation) = batch.iter().next().expect("....");
self.export_log_data(record,instruction)
```

Reply:
Nice! just pushed that change.